yr/alexa/tutorials/Number_Guessing_Game.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Guessing Game
+layout: tutorial
+howtos: false
+yrmedia: false
+---
+
+# Challenge
+<p style="font-size: 150%;
+font-weight:bold; color:#1c2f8d; padding-bottom: 0;">Make Your Own Number Guessing Game!</p>
+
+In this tutorial, we will be using the <strong>App Inventor Alexa Skills interface</strong> to create an interactive number guessing game, where the player tries to guess a random number selected by Alexa. After each guess, Alexa will give you feedback on whether your guess is too big, too small, or just right! Once you have finished this tutorial, you should understand the basics of creating Alexa Skills, along with the use of <strong>conditional statements</strong> and the <strong> Ask</strong> block.
+
+<hint markdown="block" title="What is an Alexa Skill?">
+
+An Alexa Skill is a <strong>voice-based application</strong> for Amazon Alexa-enabled devices, such as Amazon Echo Dots for Alexa-enabled smartphones. Don't worry about developing and testing Alexa Skills if you don't have your own Alexa-enabled devices. We'll show you how to use the Amazon  Developer page to test your skills using a virtual Alexa that can be run on any dvice!
+
+</hint>
+
+# Setup
+<div class="setup" id="connect_app"></div>
+
+## Create an Amazon Developer Account
```

Comment:
I think this section could be too repetitive. We should ask Selim about this. 
My thoughts are if there exist a gradual step of which tutorial you take this section is just repeating the same thing as before.